src/main/java/org/fundacionjala/coding/Yuri/BankOCR/Entry.java

```diff
@@ -0,0 +1,80 @@
+package org.fundacionjala.coding.Yuri.BankOCR;
+
+import java.util.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by YVayneR on 3/10/2017.
+ */
+public class Entry {
+    private final int SIZE = 9;
+    public String[] numbers;
+    Map<String, String> map = new HashMap<>();
+    public Entry() {
```

Comment:
one observation
Is this constructor necessary? I don't think so.
will change the map? no, because only will map 10 number 0-9.

you can define the static values.
for instance
```java
private static final Map<Integer, String> MAP = new HashMap<>();
static {
    MAP.put(" _ | ||_|","0");
    MAP.put("     |  |","1");
}
```